src/globus_cli/commands/stat.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+import uuid
+
+import click
+import globus_sdk
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import ENDPOINT_PLUS_REQPATH, command, local_user_option
+from globus_cli.termio import Field, TextMode, display
+
+STAT_FIELDS = [
+    Field("Name", "name"),
+    Field("Type", "type"),
+    Field("Last Modified", "last_modified"),
+    Field("Size", "size"),
+    Field("Permissions", "permissions"),
+    Field("User", "user"),
+    Field("Group", "group"),
+]
+
+
+@command(
+    "stat",
+    short_help="Get the status of a path",
+    adoc_examples=r"""Get the status of a path on a collection.
+
+[source,bash]
+----
+$ col_id=6c54cade-bde5-45c1-bdea-f4bd71dba2cc
+$ globus stat $col_id:/home/share/godata/file1.txt
+----
+
+""",
+)
+@click.argument("endpoint_plus_path", type=ENDPOINT_PLUS_REQPATH)
+@local_user_option
+@LoginManager.requires_login("transfer")
+def stat_command(
+    login_manager: LoginManager,
+    *,
+    endpoint_plus_path: tuple[uuid.UUID, str | None],
```

Comment:
I tried out the tests on this with the latest SDK release, and that allowed the annotation tests to run, which flagged this:
```suggestion
    endpoint_plus_path: tuple[uuid.UUID, str],
```

(Because the path is required, the type is `str`, not `str | None`.)